include/guisan/widgets/dropdown.hpp

```diff
@@ -6,11 +6,11 @@
  * /______/ //______/ //_/ //_____/\ /_/ //_/ //_/ //_/ //_/ /|_/ /
  * \______\/ \______\/ \_\/ \_____\/ \_\/ \_\/ \_\/ \_\/ \_\/ \_\/
  *
- * Copyright (c) 2004, 2005, 2006, 2007 Olof Naess�n and Per Larsson
+ * Copyright (c) 2004, 2005, 2006, 2007 Olof Naessén and Per Larsson
```

Comment:
Not sure if we use UTF-8 everywhere :-)

Reply:
Shouldn't we? I can go through the files and make sure they are all UTF-8, and using spaces instead of tabs. Might end up being lots of them in a PR :)

Reply:
I meant that you change encoding.
Not sure it is intended, probably just a default of IDE.
Anyway, I don't care.